src/generators/operationGenerator.ts

```diff
@@ -666,14 +666,9 @@ function writeNoOverloadsOperationBody(
   }
 
   const sendParams = parameterDeclarations
-    .map(p => (p.name === "options" ? `options: ${options}` : p.name))
+    .map(p => (p.name === "options" ? `options` : p.name))
```

Comment:
I think options can take the value of "updatedOptions" (line 665) with the former code we would generate `{options: updatedOptions}` this will always generate `{options}`